src/lib.ts

```diff
@@ -245,14 +245,80 @@ export async function getDestination(platform: string): Promise<string[]> {
     case 'macOS':
       return ['-destination', 'platform=macOS']
     case 'mac-catalyst':
-      return ['-destination', 'platform=macOS,variant=Mac Catalyst', 'CODE_SIGN_IDENTITY=-']
+      return ['-destination', 'platform=macOS,variant=Mac Catalyst']
     case '':
       return []
     default:
       throw new Error(`Invalid platform: ${platform}`)
   }
 }
 
+export function getIdentity(identity: string, platform: string): string | null {
+  if (identity) {
+    return `CODE_SIGN_IDENTITY="${identity}"`
+  }
+
+  if (platform == 'mac-catalyst') {
+    // Disable code signing for Mac Catalyst unless overridden.
+    return 'CODE_SIGN_IDENTITY=-'
```

Comment:
Should add an `info` so the user knows we have done this like we do for test actions with watchOS < Xcode 12.5.

Reply:
Absolutely! Will do.